hackerforms/input_types.py

```diff
@@ -108,15 +108,17 @@ def json(self):
 
 class NumberInput(Input):
     type = 'number-input'
-    def __init__(self, key: str, message: str):
+    def __init__(self, key: str, message: str, integer: bool = False):
         super().__init__(key)
         self.message = message
+        self.integer  = integer
 
     def json(self):
         return {
             'type': self.type,
             'key': self.key,
             'message': self.message,
+            'integer': self.integer,
```

Comment:
@igorctorres , on second thought, what about using `step: number` instead of `integer: bool`?